themes/openideal_theme/openideal_theme.theme

```diff
@@ -74,3 +79,16 @@ function openideal_theme_theme_suggestions_comment_alter(array &$suggestions, ar
     $suggestions[] = 'comment__view_mode__' . $elements['#view_mode'];
   }
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function openideal_theme_preprocess_field(&$variables) {
```

Comment:
Can't we add preprocess function especially for this field?
Something like `openideal_theme_preprocess_field__created()`